# *De novo* genome assembly and comparative genomics for the colonial ascidian *Botrylloides violaceus*

Jack T. Sumner [1,+] , Cassidy L. Andrasz [1,+], Christine A. Johnson [1], Sarah Wax [1], Paul Anderson [2], Elena L. Keeling [1,^], Jean M. Davidson [1,^]

[1] Department of Biological Sciences, California Polytechnic State University, San Luis Obispo, CA 93407, USA

[2] Department of Computer Science and Software Engineering, California Polytechnic State University, San Luis Obispo, CA 93407, USA

[+] These authors contributed equally to this work and share first authorship.

[^] Corresponding Authors: ekeeling@calpoly.edu, jdavid06@calpoly.edu

## Abstract
Ascidians have the potential to reveal fundamental biological insights related to coloniality, regeneration, immune function, and the evolution of these traits. This study implements a hybrid assembly technique to produce a genome assembly and annotation for the botryllid ascidian, *Botrylloides violaceus*. A hybrid genome assembly was produced using Illumina, Inc. short and Oxford Nanopore Technologies long-read sequencing technologies. The resulting assembly is comprised of 831 contigs, has a total length of 121 Mbp, N50 of 1 Mbp and a BUSCO score of 96.1%. Genome annotation identified 13K protein-coding genes. Comparative genomic analysis with other tunicates reveals patterns of conservation and divergence within orthologous gene families even among closely related species. Characterization of the Wnt gene family, encoding signaling ligands involved in development and regeneration, reveals conserved patterns of subfamily presence and gene copy number among botryllids. This supports the use of genomic data from non-model organisms in the investigation of biological phenomena.
